Packs/PaloAltoNetworks_IoT/Integrations/PaloAltoNetworks_IoT/PaloAltoNetworks_IoT.py

```diff
@@ -0,0 +1,582 @@
+
+        CommandResults
+    """
+    offset = args.get('offset', '0')
+    pagelength = args.get('limit', client.max_fetch)
+    result = client.list_devices(offset, pagelength)
+
+    outputs = {
+        'devices': result
+    }
+
+    return CommandResults(
+        outputs_prefix="PaloAltoNetworksIoT.Devices",
+        outputs_key_field='deviceid',
+        outputs=outputs
+    )
+
+
+def iot_list_alerts(client, args):
+    """
+    Returns a list of IoT alerts (max: 1000)
+
+    Args:
+        client (Client): IoT client.
+        args (dict): all command arguments.
+
+    Returns:
+        List of alerts
+
+        CommandResults
+    """
+    stime = args.get('start_time', '-1')
+    offset = args.get('offset', 0)
+    pagelength = min(int(args.get('limit', client.max_fetch)), PAGELENGTH)
+    result = client.list_alerts(stime, offset, pagelength, 'desc')
+
+    outputs = {
+        'alerts': result
+    }
+
+    return CommandResults(
+        outputs_prefix="PaloAltoNetworksIoT.Alerts",
+        outputs_key_field="id",
+        outputs=outputs
+    )
+
+
+def iot_list_vulns(client, args):
+    """
+    Returns a list of IoT vulnerabilties (max: 1000)
+
+    Args:
+        client (Client): IoT client.
+        args (dict): all command arguments.
+
+    Returns:
+        List of vulnerabilties
+
+        CommandResults
+    """
+    stime = args.get('start_time', '-1')
+    offset = args.get('offset', 0)
+    pagelength = min(int(args.get('limit', client.max_fetch)), PAGELENGTH)
+    result = client.list_vulns(stime, offset, pagelength)
+
+    outputs = {
+        'vulns': result
+    }
+
+    return CommandResults(
+        outputs_prefix="PaloAltoNetworksIoT.Vulns",
+        outputs_key_field="zb_ticketid",
+        outputs=outputs
+    )
+
+
+def iot_resolve_alert(client, args):
+    """
+    Resolve an IoT alert
+
+    Args:
+        client (Client): IoT client.
+        args (dict): all command arguments.
+
+    Returns:
+        None in CommandResults
+    """
+    alert_id = args.get('id')
+    reason = args.get('reason', 'resolved by XSOAR')
+    reason_type = args.get('reason_type', 'No Action Needed')
+
+    result = client.resolve_alert(alert_id, reason, reason_type)
+
+    return CommandResults(
+        readable_output=f'{json.dumps(result)}',
+        outputs_prefix="",
+        outputs_key_field="",
+        outputs=None
+    )
+
+
+def iot_resolve_vuln(client, args):
+    """
+    Resolve an IoT vulnerability
+
+    Args:
+        client (Client): IoT client.
+        args (dict): all command arguments.
+
+    Returns:
+        None in CommandResults
+    """
+    vuln_id = args.get('id')
+    full_name = args.get('full_name')
+    reason = args.get('reason', 'resolved by XSOAR')
+
+    result = client.resolve_vuln(vuln_id, full_name, reason)
+
+    return CommandResults(
+        readable_output=f'{json.dumps(result)}',
+        outputs_prefix="",
+        outputs_key_field="",
+        outputs=None
+    )
```

Comment:
```suggestion
    return CommandResults(
        readable_output= vulnerability {vuln_id} was resolved successfully',
    )
```

Reply:
`
        PaloAltoNetworks_IoT.py:258:15: E1120: No value for argument 'outputs_prefix' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:258:15: E1120: No value for argument 'outputs_key_field' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:258:15: E1120: No value for argument 'outputs' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:286:15: E1120: No value for argument 'outputs_prefix' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:286:15: E1120: No value for argument 'outputs_key_field' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:286:15: E1120: No value for argument 'outputs' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:314:15: E1120: No value for argument 'outputs_prefix' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:314:15: E1120: No value for argument 'outputs_key_field' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:314:15: E1120: No value for argument 'outputs' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:340:11: E1120: No value for argument 'outputs_prefix' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:340:11: E1120: No value for argument 'outputs_key_field' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:340:11: E1120: No value for argument 'outputs' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:362:11: E1120: No value for argument 'outputs_prefix' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:362:11: E1120: No value for argument 'outputs_key_field' in constructor call (no-value-for-parameter)
        PaloAltoNetworks_IoT.py:362:11: E1120: No value for argument 'outputs' in constructor call (no-value-for-parameter)`

so I am adding back those and keep the readable

Reply:
No need, let's speak on private